src/styles/_catalog.scss

```
@@ -20,3 +21,28 @@
     margin-right: 48px;
   }
 }
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .catalog {
+    grid-template-columns: repeat(3, 1fr);
```

Also here

```suggestion
    grid-template-columns: repeat(3, auto);
```

And in all other places below

---

Виконала